man/passwd.1.xml

```diff
@@ -65,60 +65,61 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
     <para>
-      The <command>passwd</command> command changes passwords for user accounts.
-      A normal user may only change the password for their own account, while
-      the superuser may change the password for any account.
-      <command>passwd</command> also changes the account or associated
-      password validity period.
+      The <command>passwd</command> command changes passwords
+      for user accounts.
+      A regular user can only change the password for their own account,
+      while the superuser can change the password for any account.
+      The <command>passwd</command> also changes the account
+      or associated password validity period.
     </para>
 
     <refsect2 id='password_changes'>
       <title>Password Changes</title>
       <para>
-	The user is first prompted for their old password, if one is
-	present. This password is then encrypted and compared against the
-	stored password. The user has only one chance to enter the correct
-	password. The superuser is permitted to bypass this step so that
-	forgotten passwords may be changed.
+	If the account has a non-empty password,
+	the user is first prompted to enter their current password.
+	The entered password is encrypted and compared to the stored value.
+	The user has only one attempt to enter the correct password.
+	The superuser can bypass this step to allow changing forgotten passwords.
       </para>
 
       <para>
 	After the password has been entered, password aging information is
-	checked to see if the user is permitted to change the password at
+	checked to determine if the user is permitted to change the password at
 	this time. If not, <command>passwd</command> refuses to change the
 	password and exits.
       </para>
 
       <para>
 	The user is then prompted twice for a replacement password. The
 	second entry is compared against the first and both are required to
-	match in order for the password to be changed.
+	match for the password to be changed.
       </para>
 
       <para>
 	Then, the password is tested for complexity.
-	<command>passwd</command> will reject any password which is not
-	suitably complex.  Care must be taken not to include the system
-	default erase or kill characters.
+	<command>passwd</command> rejects passwords that do not meet
+	the complexity requirements.
+	Do not include the system default erase or kill characters.
```

Comment:
Not a question about the documentation change, but about the old text.

@ikerexxe , @hallyn  is this a warning to not introduce bad characters in passwords?  Should we add checks like we have for the user name?  `Care must be taken` sounds like it will break something in `/etc/passwd`, which sounds like we should check.